src/api/db/schema.rb

```diff
@@ -425,8 +425,8 @@
     t.index ["workflow_run_id"], name: "index_event_subscriptions_on_workflow_run_id"
   end
 
-  create_table "events", id: :integer, charset: "utf8mb4", collation: "utf8mb4_unicode_ci", options: "ENGINE=InnoDB ROW_FORMAT=DYNAMIC", force: :cascade do |t|
-    t.string "eventtype", null: false, collation: "utf8_unicode_ci"
+  create_table "events", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", options: "ENGINE=InnoDB ROW_FORMAT=DYNAMIC", force: :cascade do |t|
+    t.string "eventtype", null: false, collation: "utf8mb3_unicode_ci"
```

Comment:
The id parameter disappeared from this line. Is this normal?

I checked in `dbconsole` after running the migration and  the `bigint` is there:

```
MariaDB [api_development]> describe events;                                                                                                    
+-------------+--------------+------+-----+---------+----------------+
| Field       | Type         | Null | Key | Default | Extra          |
+-------------+--------------+------+-----+---------+----------------+
| id          | bigint(20)   | NO   | PRI | NULL    | auto_increment |
| eventtype   | varchar(255) | NO   | MUL | NULL    |                |
| payload     | text         | YES  |     | NULL    |                |
| created_at  | datetime     | YES  | MUL | NULL    |                |
| updated_at  | datetime(6)  | YES  |     | NULL    |                |
| undone_jobs | int(11)      | YES  |     | 0       |                |
| mails_sent  | tinyint(1)   | YES  | MUL | 0       |                |
+-------------+--------------+------+-----+---------+----------------+

```

Reply:
> The id parameter disappeared from this line. Is this normal?

What happens if you create a new table?